cypress/support/step_definitions/index.ts

```diff
@@ -58,19 +67,38 @@ When('I wait for {int} milliseconds/ms', function (ms: number) {
   cy.wait(ms)
 })
 
-When(/^I "(.*)"(.*)? on the "(.*)" element$/, (event: string, object: string | number | undefined, selector: string) => {
-  switch (event) {
-    case 'select':
-      if (typeof object === 'undefined') {
-        throw new Error()
-      }
-      $(selector).select(parseInt(object.toString()), { force: true })
-      break
-    default:
-      $(selector).trigger(event, { force: true })
+When(/^I click the "(.*)" element(?: and select "(.*)")?$/, (selector: string, value?: string) => {
```

Comment:
Ok can we leave the "" in for the moment please? This PR is about adding a zone creation view, not discussing already existing patterns and I think we are getting very far away from the task here. If you want to change this then we should discuss separately and make a decision that way, otherwise all of this is needlessly holding up your PR, considering the original PR didn't have this change in it at first, it's these changes here that are holding it up.

If you want to discuss these little details further let me know. I'd be happy to chat through again.

Reply:
You suggested very strongly and explicitly to keep things as simple as possible when it comes to these step utilities and allow as little as possible variety in how they can be used. In this interest, I applied several of your suggestions beyond what I had initially presented in this PR.

So to say that me deviating from the core of this PR’s task is what’s holding up the PR is getting it backwards: you suggested this work.

Is it really necessary to block this PR over what you call a little detail that has absolutely no functional consequence and is really a matter of taste?

Reply:
I’m resolving this because it’s purely a matter of taste and was only changed on detailed suggestion.